oidc-controller/src/VCAuthn/IdentityServer/IdentityConstants.cs

```diff
@@ -46,6 +46,7 @@ public class IdentityConstants
         
         public const string ChallengeIdQueryParameterName = "pid";
         public const string ChallengePollUri = "vc/connect/poll";
+        public const int PollInterval = 2000;
```

Comment:
Any way we could make this configurable by reading an environment variable and fall back to the default value if that is not set?

I like the idea, however this requires the app to be rebuilt in order for the change to take effect.

Reply:
i've changed so that poll interval can be configured in appsettings.json and env var in manage script